ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -97,30 +97,30 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-ovn-fips-p2-f7
-  cron: 42 6 16,24,31,7 * *
+- as: aws-ipi-ingress-lbtype-nlb-p2-f7
```

Comment:
This file is upgrade-from-stable-4.11, but we don't support `ingress-lbtype-nlb` in 4.11.

Reply:
Good catch. Reverted the change for this file.